lib/cli-engine.js

```diff
@@ -298,23 +301,31 @@ function getCacheFile(cacheFile, cwd) {
     }
 
 
-    // in case the file exists we need to verify if the provided path
-    // is a directory or a file. If it is a directory we want to create a file
-    // inside that directory
+    /*
+     * in case the file exists we need to verify if the provided path
+     * is a directory or a file. If it is a directory we want to create a file
+     * inside that directory
+     */
     if (fileStats) {
-        // is a directory or is a file, but the original file the user provided
-        // looks like a directory but `path.resolve` removed the `last path.sep`
-        // so we need to still treat this like a directory
+
+        /*
+         * is a directory or is a file, but the original file the user provided
+         * looks like a directory but `path.resolve` removed the `last path.sep`
+         * so we need to still treat this like a directory
+         */
```

Comment:
@mysticatea You'll want to read the issue and the Coding Conventions
document- that's what I'm following with these changes. I agree with you to
an extent, but the fact is @nzakas drafted a set of coding guidelines which
aren't being followed and aren't being enforced by ESLint. If you want to
propose a change to the guidelines, the issue is a great place to do it.

Regarding a blank line between open parentheses and the next token, I
prefer to think of it as a sign that the expression within the parentheses
is too complex or abstract and should be extracted to a suitably named
function so that it becomes self-explanatory. :smile:
On Mar 19, 2016 3:29 PM, "Toru Nagashima" <notifications@github.com> wrote:

> In lib/cli-engine.js
> <https://github.com/eslint/eslint/pull/5622#discussion_r56754643>:
>
> >      if (fileStats) {
> > -        // is a directory or is a file, but the original file the user provided
> > -        // looks like a directory but `path.resolve` removed the `last path.sep`
> > -        // so we need to still treat this like a directory
> > +
> > +        /*
> > +         * is a directory or is a file, but the original file the user provided
> > +         * looks like a directory but `path.resolve` removed the `last path.sep`
> > +         * so we need to still treat this like a directory
> > +         */
>
> Hmmm, the empty line between open parentheses and comments looks strange
> to me....
> And this is just my feeling, I like line-comments than block-comments,
> since only /* or */ lines.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/pull/5622/files/4b05ce6482d0d93a798369997618d2adc5ea15e1#r56754643>
>